experimental/InvariantTheory/secondary_invariants.jl

```diff
@@ -217,10 +217,22 @@ function secondary_invariants_modular(RG::InvRing)
   s_invars_cache = SecondaryInvarsCache{elem_type(Rgraded)}()
   add_invariant!(s_invars_cache, one(Rgraded), false, Int[])
 
+  # The maximal degree in which we have to look for secondary invariants by Sym11.
```

Comment:
I was confused for a moment why a symmetric group of degree 11 comes up here ;-).

Perhaps like this:
```suggestion
  # The maximal degree in which we have to look for secondary invariants by [Sym11].
```
or
```suggestion
  # The maximal degree in which we have to look for secondary invariants by \cite{Sym11}.
```